src/components/Textarea/index.js

```diff
@@ -1,12 +1,14 @@
 import React, { Component } from 'react';
 import PropTypes from 'prop-types';
-import classnames from 'classnames';
 import autosize from 'autosize';
 import withReduxForm from './../../libs/hocs/withReduxForm';
 import { uniqueId } from './../../libs/utils';
 import RenderIf from '../RenderIf';
 import Label from './label';
-import './styles.css';
+import StyledContainer from './styled/container';
+import StyledTextarea from './styled/textarea';
+import StyledBottomHelp from './styled/bottomHelp';
+import StyledError from './styled/error';
```

Comment:
bottomHelp and error are already defined in input component so use these instead:
```input/styled/helpText```
```input/styled/errorText```
same for label, use this one:
```input/label```
also remove label and his test file and all other styeld files